golem-common/src/model/mod.rs

```diff
@@ -585,18 +585,10 @@ pub struct WorkerStatusRecord {
     pub total_linear_memory_size: u64,
     pub owned_resources: HashMap<WorkerResourceId, WorkerResourceDescription>,
     pub oplog_idx: OplogIndex,
-    pub extensions: WorkerStatusRecordExtensions,
-}
-
-#[derive(Clone, Debug, PartialEq, Encode, Decode)]
-pub enum WorkerStatusRecordExtensions {
```

Comment:
As this can be recomputed from the oplog I decided to go that way (as I need that anyway for component_version_for_replay)